packages/core/src/html/NodeProcessor.ts

```diff
@@ -268,9 +305,9 @@ class NodeProcessor {
     }
   }
 
-  _process(node, context) {
+  private traverse(node: DomElement, context: Context): DomElement | DomElement[] {
     if (_.isArray(node)) {
```

Comment:
From what I can see, it seems like this check is not necessary since `node` will not be an array? If that is the case, we can remove this block and `traverse` should just return a `DomElement`.